src/styles/main.scss

```diff
@@ -0,0 +1,61 @@
+$day-size: 100px;
+$gap-size: 1px;
+$columns: 7;
+$padding-size: 10px;
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  padding: $padding-size;
+  gap: $gap-size;
+  max-width: calc(
+    #{$columns} * #{$day-size} + (#{$columns} - 1) * #{$gap-size} + 2 * #{$padding-size}
+  );
+
+  .day {
+    display: flex;
+    justify-content: center;
+    box-sizing: border-box;
+    height: $day-size;
+    width: $day-size;
+    background-color: #eee;
+    border: 1px solid black;
+    font-family: Arial, Helvetica, sans-serif;
```

Comment:
The font stack should include more generic family names as a fallback, such as `serif` or `sans-serif`. This is a recommendation from the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules) to ensure better cross-platform compatibility.